bluecellulab/graph.py

```diff
@@ -0,0 +1,85 @@
+"""Graph representation of Cells and Synapses."""
+
+from matplotlib.cm import ScalarMappable
+from matplotlib.colors import Normalize
+import matplotlib.pyplot as plt
+import networkx as nx
+import numpy as np
+
+
+from bluecellulab.cell.cell_dict import CellDict
+
+
+def build_graph(cells: CellDict) -> nx.DiGraph:
+    G = nx.DiGraph()
+
+    # Add nodes (cells) to the graph
+    for cell_id, cell in cells.items():
+        G.add_node(cell_id, label=str(cell_id.id), population=cell_id.population_name)
```

Comment:
As far as I can tell, when fetching the population_name from G afterwards, population_name is not used, but cell_id.population_name.
Is it useful to add a population_name to G if you only fetch it from G.cell_id?

Reply:
While plotting the plot in the later step the population name gives the color of the node and it is read from G.